.flake8

```diff
@@ -0,0 +1,9 @@
+[flake8]
+exclude = 
+    .git
+    ,.circleci/run-clang-format.py
```

Comment:
@blefaudeux this particular file has a MIT license, so I added an exclude here, pls let me know if this should not be skipped.